datafusion/ffi/src/execution_plan.rs

```diff
@@ -372,6 +379,51 @@ mod tests {
 
         assert!(original_name == foreign_plan.name());
 
+        let display = datafusion::physical_plan::display::DisplayableExecutionPlan::new(
+            &foreign_plan,
+        );
+
+        let mut buf = String::new();
+        write!(&mut buf, "{}", display.one_line()).unwrap();
+        let buf = buf.trim();
```

Comment:
I think you can probably do this more succintly using `to_string()`

Something like this (untested):
```suggestion
        let buf = display.one_line().to_string().trim();
```